book/MCDM Productions; Arcadia Issue 8.json

```diff
@@ -0,0 +1,3593 @@
+{
+    "$schema": "https://raw.githubusercontent.com/TheGiddyLimit/homebrew/master/_schema-fast/homebrew.json",
```

Comment:
The `$schema` key is VSCode-exclusive, and while it's very useful during testing it's better to remove them before uploading to the repo
```suggestion
```